src/NLog/Layouts/JsonLayout.cs

```diff
@@ -80,9 +80,9 @@ public bool SerializeObject(object value, StringBuilder builder)
                     return _converter.SerializeObject(value, builder);
             }
 
-            public bool SerializeObjectNoLimit(object value, StringBuilder builder)
+            public void SerializeObjectNoLimit(object value, StringBuilder builder)
```

Comment:
You are touching public methods. Breaking change.

Reply:
Nope just ignore me

Reply:
👼 That's the reason I don't like the keyword `public` in internal types ;)